test/JDBC/parallel_query_jdbc_schedule

```diff
@@ -345,6 +326,8 @@ ignore#!#getdate-vu-prepare
 ignore#!#getdate-vu-verify
 ignore#!#getdate-vu-cleanup
 ignore#!#getdatetest
+ignore#!#TestDatetime-numeric-representation-vu-prepare
```

Comment:
why are these ignored?

Reply:
These tests are timing out. So we will take it separately.